docs/tutorial.md

```diff
@@ -426,10 +428,22 @@ with open("example_snps.vcf", "w") as vcffile:
     ts.write_vcf(vcffile, individuals=keep_indivs)
 ```
 
+If you've done nothing else to the output from SLiM,
+then this code will work,
+but it does requires all alive individuals to be *samples*.
+A situation in which this isn't the case is shown in the next section.
+
 
 ### Extracting individuals after simplification
 
-In cases where we simplify, we need to pay attention to specific node IDs.
```

Comment:
I tried to clarify what "pay attention to node IDs" means here. Also, I'm trying not to say "sample individuals", since "being a sample" is a property of a node, not an individual,  technically.